src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -48,6 +48,8 @@ ContextAttribute::ContextAttribute()
   value                 = "";
   compoundValueP        = NULL;
   typeFromXmlAttribute  = "";
+  providingApplication  = "";
+  found                 = true;
```

Comment:
By default, `found` is set to true. I think I have touched all constructors, so no "randomness" will happen. However, it woudl be great @kzangeli if you could double-check is ok.